core/domain/platform_parameter_domain.py

```diff
@@ -493,15 +597,15 @@ def filters(self):
         return self._filters
 
     @property
-    def value_when_matched(self):
+    def value_when_matched(self) -> Union[str, bool, int]:
```

Comment:
Should we save `Union[str, bool, int]` int oa variable? You seem to be using it in multiple places.

Reply:
Done